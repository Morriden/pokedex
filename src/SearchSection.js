import React, { Component } from 'react'

export default class SearchSection extends Component {
    render() {
        return (
            
                <div className="search-area"> 
                        
                <input onChange={this.props.CallBackhandleChange}/>
                <button onClick={this.props.CallBackhandleClick}>Search</button>  
                <select onChange={this.props.CallBackhandleDropDownChange}>
                    <option value="pokemon">Filter by Pokemon</option>
                    <option value="type">Filter by Type</option>
                    <option value="attack">Filter by Attack</option>
                    <option value="defense">Filter by Defense</option>
                </select> 
                </div>
            
        )
    }
}