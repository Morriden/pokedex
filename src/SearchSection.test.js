import React from 'react';
import SearchSection from './SearchSection.js';
import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';

test('will come back without stuff', () => {
    const wrapper = shallow(<SearchSection />)

   expect(wrapper).toMatchSnapshot();

});

test('will come back with stuff', () => {
    const wrapper = shallow(<SearchSection name="some name"/>)

   expect(wrapper).toMatchSnapshot();

});