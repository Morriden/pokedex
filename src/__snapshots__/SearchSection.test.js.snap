// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will come back with stuff 1`] = `
<div
  className="search-area"
>
  <input />
  <button>
    Search
  </button>
  <select>
    <option
      value="pokemon"
    >
      Filter by Pokemon
    </option>
    <option
      value="type"
    >
      Filter by Type
    </option>
    <option
      value="attack"
    >
      Filter by Attack
    </option>
    <option
      value="defense"
    >
      Filter by Defense
    </option>
  </select>
</div>
`;

exports[`will come back without stuff 1`] = `
<div
  className="search-area"
>
  <input />
  <button>
    Search
  </button>
  <select>
    <option
      value="pokemon"
    >
      Filter by Pokemon
    </option>
    <option
      value="type"
    >
      Filter by Type
    </option>
    <option
      value="attack"
    >
      Filter by Attack
    </option>
    <option
      value="defense"
    >
      Filter by Defense
    </option>
  </select>
</div>
`;
